package.json

```diff
@@ -119,12 +119,12 @@
     "build:examples": "cd examples && node buildAll.js",
     "pretest": "yarn lint",
     "prelint": "yarn setup",
-    "lint": "yarn code-lint && yarn jest-lint && yarn type-lint && yarn special-lint",
-    "code-lint": "eslint --cache \"{setup,lib,bin,hot,buildin,benchmark,tooling,schemas}/**/*.js\" \"test/*.js\" \"test/{configCases,watchCases,statsCases,hotCases}/**/webpack.config.js\" \"examples/**/webpack.config.js\"",
+    "lint": "yarn eslint && yarn jest-lint && yarn type-lint && yarn special-lint",
+    "eslint": "eslint . --ext '.js'",
```

Comment:
rename it back to `code-lint`. `eslint` would collide with running `eslint` inself.

Reply:
Please check the performance implication of this. As far as I remember this would scan the whole directory and `test/js` can have many files after normal test runs.